ikarus/finiteelements/mechanics/kirchhoffloveshell.hh

```diff
@@ -36,18 +37,19 @@ namespace Ikarus {
   template <typename Basis_, typename FERequirements_ = FERequirements<>, bool useEigenRef = false>
   class KirchhoffLoveShell : public PowerBasisFE<typename Basis_::FlatBasis> {
   public:
-    using Basis                             = Basis_;
-    using FlatBasis                         = typename Basis::FlatBasis;
-    using BasePowerFE                       = PowerBasisFE<FlatBasis>;  // Handles globalIndices function
-    using FERequirementType                 = FERequirements_;
-    using ResultRequirementsType            = ResultRequirements<FERequirementType>;
-    using LocalView                         = typename FlatBasis::LocalView;
-    using Element                           = typename LocalView::Element;
-    using Geometry                          = typename Element::Geometry;
-    using GridView                          = typename FlatBasis::GridView;
-    using Traits                            = TraitsFromLocalView<LocalView, useEigenRef>;
-    static constexpr int myDim              = Traits::mydim;
-    static constexpr int worlddim           = Traits::worlddim;
+    using Basis                   = Basis_;
```

Comment:
Do you know what happened here? Are there trailing white spaces or something=?

Reply:
Here, it is now not aligning as per the `=` in the following lines:
```cpp
static constexpr int membraneStrainSize = 3;
static constexpr int bendingStrainSize  = 3;
```
It may be because I added this line that is very long and it is not refactoring it into two lines
```cpp
using LocalBasisType          = decltype(std::declval<LocalView>().tree().child(0).finiteElement().localBasis());
```
There are no trailing white spaces, but I do not completely understand the formatting mechanism at this point here

Reply:
For readability, I added an empty line in between

Reply:
Ah I see the equal sign is now one space to the right